src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -1670,9 +1678,27 @@ neg-distrib-⊓-⊔ = antimono-≤-distrib-⊓ neg-mono-≤
   ℤ.+ n ℤ.* 1ℤ       ∎)
   where open ℤ.≤-Reasoning
 
-∣p∣≡p∨∣p∣≡-p : ∀ p → (∣ p ∣ ≡ p) ⊎ (∣ p ∣ ≡ - p)
-∣p∣≡p∨∣p∣≡-p (mkℚᵘ (ℤ.+ n)    d-1) = inj₁ refl
-∣p∣≡p∨∣p∣≡-p (mkℚᵘ (-[1+ n ]) d-1) = inj₂ refl
+∣p∣≡p⊎∣p∣≡-p : ∀ p → (∣ p ∣ ≡ p) ⊎ (∣ p ∣ ≡ - p)
```

Comment:
Nice spot! Although actually this name was correct and the other name is wrong, as we use logical operators such as `∨` in names rather than type-level operators such as `⊎` (see the [style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#preconditions-and-postconditions)).

Would it be possible to change this one back, and then change the one in the normalised rationals to match?

Reply:
Fixed, and altered the one in `Data.Rational.Properties`.